### PR TITLE
Issue 801 execution quickrun

### LINKED ISSUE #801: User guide: Add execution quickrun page (primary page)

### PR DESCRIPTION
**Pull request by richardjgowers**
fixes #801

Checklist
* [ ] Added a ``news`` entry

## Developers certificate of origin
- [ ] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

File: docs/guide/execution/quickrun_execution.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Execution with Quickrun
+=======================
+
+The planning and preparation of a campaign of alchemical simulations using the ``openfe`` package is intended to be
+achievable on a local workstation in a matter of minutes.
+The **execution** of these simulations however requires a large amount of computational power and is intended to be
+distributed onto a HPC environment.
+To do this requires storing and sending the details of the simulation from the local workstation to a HPC environment,
+this can be done via the :func:`.Transformation.dump` function which creates a saved "json" version of the data.
```
Comment (IAlibay) [inline docs/guide/execution/quickrun_execution.rst:11]: Should this really be here? It feels like this page should concentrate "I have a JSON, what do I do now", and this should get punted over to the setup section.. maybe we're missing a "writing out an alchemical network" section?

**Review by IAlibay — CHANGES_REQUESTED** [review]
Thanks, couple of typos and a question.

File: docs/guide/execution/quickrun_execution.rst (hunk 1 above)
Comment (richardjgowers) [reply docs/guide/execution/quickrun_execution.rst]: I added a link to the relevant cookbook here.  It's duplicated information, but it's also essential for anything else on the page to make sense so it's probably warranted

File: docs/guide/execution/quickrun_execution.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.. _userguide_quickrun:
+
+Execution with Quickrun
+=======================
+
+The planning and preparation of a campaign of alchemical simulations using the ``openfe`` package is intended to be
+achievable on a local workstation in a matter of minutes.
+The **execution** of these simulations however requires a large amount of computational power and is intended to be
+distributed across a HPC environment.
```
Comment (IAlibay) [inline docs/guide/execution/quickrun_execution.rst:9]: It's probably not necessary, but one thing here that might be a bit of a catch to a novice user is "o so I need a HPC machine to do this", maybe a note that says something like "you can also do this on your workstation" might be good to spell things out?

File: docs/guide/execution/quickrun_execution.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+and contain all the information required to execute a single simulation.
+
+To read this information and execute the simulation, the command line interface provides a ``quickrun`` command,
+the full details of which are given in :ref:`the CLI reference section<cli_quickrun>`.
+Briefly, this command takes a "json" simulation as an input and will then execute the simulation contained within,
+therefore this command would execute a simulation saved to a file called "transformation.json".
+
+::
+
+  openfe quickrun transformation.json
```
Comment (IAlibay) [inline docs/guide/execution/quickrun_execution.rst:23]: This doesn't use the `-o` flag, but the next example does. Would it be worth at least making them equivalent? (and saying that -o means that you're generating an output file named results.json)?

**Review by IAlibay — APPROVED** [review]
Two comments, but otherwise looks good to me.

**Pull request merged**